src/main/java/com/google/sps/servlets/ColorServlet.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.io.IOException;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+
+@WebServlet("/color")
+public class ColorServlet extends HttpServlet {
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) 
+    throws IOException {
+    
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    String username;
+    String tone;
+
+    UserService userService = UserServiceFactory.getUserService();
+
+    if (!userService.isUserLoggedIn()) {
+      return;
+    }
+
+    username = userService.getCurrentUser().getEmail();
+    tone = request.getParameter("tone");
+
+    //Final Entity will be UserData. This section begins to populate the database
+    Entity ColorTest1 = new Entity("ColorTest1");
```

Comment:
I do not want to put the real name of the entity until all testing has been done, because if an accident is made on the entry of data, it is difficult to go and delete null entries as they can cause errors